aea/test_tools/test_cases.py

```diff
@@ -157,3 +199,20 @@ def run_install(self):
         :return: None
         """
         self.run_cli_command("install")
+
+
+class AEAWithOefTestCase(AEATestCase):
+    @pytest.fixture(autouse=True)
+    def _start_oef_node(self, network_node):
+        """Start an oef node."""
+
+    def run_oef_subprocess(self):
```

Comment:
misleading method name, you're not running the oef, you're running an agent with an oef connection.